src/main/java/com/notably/logic/parser/NewCommandParser.java

```diff
@@ -34,29 +33,26 @@
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_TITLE, PREFIX_BODY, PREFIX_JUMP);
 
-        if (!NotablyParser.arePrefixesPresent(argMultimap, PREFIX_TITLE, PREFIX_BODY)
+        if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_TITLE, PREFIX_BODY)
                 || !argMultimap.getPreamble().isEmpty()) {
             throw new ParseException(String.format("Invalid Command"));
         }
 
         String title = argMultimap.getValue(PREFIX_TITLE).get();
         String body = argMultimap.getValue(PREFIX_BODY).get();
-        try {
-            RelativePath relativePath = RelativePath.fromString(title);
-            Block block = new BlockImpl(new Title(title), new Body(body));
-            List<Command> commands = new ArrayList<>();
-            commands.add(new NewCommand(block, relativePath));
 
-            if (!NotablyParser.arePrefixesPresent(argMultimap, PREFIX_TITLE, PREFIX_BODY, PREFIX_JUMP)) {
-                return commands;
-            }
+        Path path = ParserUtil.createPath(title);
+        Block block = new BlockImpl(new Title(title), new Body(body));
+        List<Command> commands = new ArrayList<>();
+        commands.add(new NewCommand(block, path));
 
-            commands.add(new OpenCommand(relativePath));
+        if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_TITLE, PREFIX_BODY, PREFIX_JUMP)) {
```

Comment:
Nitpicking here. I think you can simply check `ParserUtil.arePrefixesPresent(argMultimap, PREFIX_JUMP)` right?